config/examples/backend_redis.yml

```
@@ -4,8 +4,19 @@ base: &default
   size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   sentinels: "<%= ENV['BACKEND_REDIS_SENTINEL_HOSTS'] %>"
+  sentinel_username: <%= ENV['BACKEND_REDIS_SENTINEL_USERNAME'].to_json %>
+  sentinel_password: <%= ENV['BACKEND_REDIS_SENTINEL_PASSWORD'].to_json %>
   name: <%= ENV['BACKEND_REDIS_SENTINEL_NAME'] %>
   role: <%= ENV['BACKEND_REDIS_SENTINEL_ROLE'] %>
+  # == ACL Params ==
+  username: <%= ENV['BACKEND_REDIS_USERNAME'].to_json %>
+  password: <%= ENV['BACKEND_REDIS_PASSWORD'].to_json %>
+  # == TLS Params ==
+  ssl: <%= ENV.fetch('BACKEND_REDIS_SSL', '0') == '1' %>
+  ssl_params:
+    ca_file: <%= ENV['BACKEND_REDIS_CA_FILE'] %>
+    cert: <%= ENV['BACKEND_REDIS_CLIENT_CERT'] %>
+    key: <%= ENV['BACKEND_REDIS_PRIVATE_KEY'] %>
```

If my assumption is correct, one can enable ssl with just CA when client auth is not needed. So we should still allow cert and key to be omitted.

Also I believe we should allow CA_FILE to be `nil` when the default operating system CA is suitable for the connection.

```suggestion
    ca_file: <%= ENV['BACKEND_REDIS_CA_FILE'].to_json %>
    cert: <%= ENV['BACKEND_REDIS_CLIENT_CERT'].to_json %>
    key: <%= ENV['BACKEND_REDIS_PRIVATE_KEY'].to_json %>
```